content/docs/command-reference/exp/show.md

```diff
@@ -62,8 +62,9 @@ metric or param.
   the workspace. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
-  as well as the workspace. This prints all experiments available.
+- `-A`, `--all-commits` - same as `-a` or `-T` above, but applies to _all_
+  reachable Git commits as well as the workspace. This prints all experiments
+  available.
```

Comment:
Oops I forgot to roll this one back 🤦

Reply:
Noted in https://github.com/iterative/dvc.org/pull/2325#discussion_r600038175 so I don't forget to roll back if needed.